.travis.yml

```diff
@@ -4,5 +4,10 @@ jdk:
 before_install:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"
+
+script:
+  - mvn javadoc:jar
+  - mvn javadoc:test-aggregate
```

Comment:
Yeah, it does that to "install dependencies".

http://docs.travis-ci.com/user/languages/java/